public/external-text/default.yaml

```diff
@@ -744,30 +769,39 @@ tabs:
 
       - label: Team
         cards:
-          - header: '[Trevor Murdock](https://pacificclimate.org/about-pcic/people/trevor-murdock)'
```

Comment:
Trevor should still be credited here on P2A. You can remove the link to his bio, and credit him with being the "Founding concept designer".